### PR TITLE
Image previews

### PR DESCRIPTION
**Pull request by tejasmanohar**
Solves #390 

here's a preview
preview

### CODE REVIEW

**Comment by tejasmanohar** [comment]
cc @tpetr

**Comment by tejasmanohar** [comment]
normal files still work btw
preview

File: SingularityUI/app/views/tail.coffee [hunk 1]
```diff
@@ -40,52 +40,68 @@ class TailView extends View
             @$el.html "<h1>Could not get request file.</h1>"
 
     render: =>
-        breadcrumbs = utils.pathToBreadcrumbs @path
+        @collection.fetchMagicString().done (res) =>
+            isImage = /PNG|ÿØÿà|GIF89a|GIF87a/.test(res.data)
```
Comment (simplyianm) [inline SingularityUI/app/views/tail.coffee]: What's `ÿØÿà`?

Comment (tejasmanohar) [inline SingularityUI/app/views/tail.coffee]: @simplyianm That's a JPEG's signature, http://en.wikipedia.org/wiki/Magic_number_%28programming%29

**Comment by ssalinas** [comment]
closing in favor of #632

**Pull request closed without merging**